all_students_rendering.php

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="lt">
+    <head>
+        <meta charset="UTF-8">
+        <title>MySQL dienynas</title>
+
+        <style>
+            table, th, td {
+                border: 1px solid black;
+                border-collapse: collapse;
+                text-align: center;
+                padding: 15px;
+            }
+        </style>
+    </head>
+    <body>
+        <table>
+            <tr>
+                <th> Mokinio ID </th>
+                <th> Vardas </th>
+                <th> Pavardė </th>
+            </tr>
+            <?php
+            for ($i = 0; $i < $counter; $i++) {
+                ?>
+                <tr>
+                    <td><?php echo $students[$i][0]; ?></td>
```

Comment:
Pabandyk perrašyti pasinaudojant `<?=` (https://stackoverflow.com/questions/2020445/what-does-mean-in-php)